docs/source/roadmaps/may-2024.md

```diff
@@ -0,0 +1,117 @@
+# May 2024
+This roadmap is split into "themes" which roughly mirror our current priorities.
+
+## Data management
+**Current status**
+
+We have developed [NeuroBlueprint](https://neuroblueprint.neuroinformatics.dev/), a simple, standard data folder 
+specification for (systems) neuroscience and [datashuttle](datashuttle.neuroinformatics.dev), to automate the 
+creation, validation and transfer of NeuroBlueprint projects. For more details on this specific project, see the 
+[datashuttle roadmap]().
+
+**Q4 2024**
+
+Standardised metadata support
+
+**Q2 2025**
+
+Support for customising data structure (e.g. specific mandatory elements or new file types)
+
+**Q4 2025**
+
+Export of NeuroBlueprint projects to [Neurodata Without Borders](https://www.nwb.org/) format.
+
+## Behavioural analysis
+**Current status**
+
+We are building [movement](https://movement.neuroinformatics.dev/) a Python toolbox for analysis of animal tracking 
+data. For more details on this specific project, see the [movement roadmap](https://movement.neuroinformatics.dev/community/roadmap.html).
+
+**Q3 2025**
+
+- Support all common post pose-estimation analysis tasks
+- Support for single experiments and complex multi-animal, multi-session datasets 
+(by reading and writing to [NeuroBlueprint](https://neuroblueprint.neuroinformatics.dev/) format)
+
+**No current ETA**
+
+- A common interface to machine-learning based video behavioural analysis tools
```

Comment:
To a non-expertm these two points seem nearly identical. Maybe we should rephrase the first one to be more specific to motion captura/tracking (or even restrict to pose estimation)? @sfmig

Reply:
Now updated to:
>- A common interface to common-pose estimation tools, and other machine-learning based object detection methods
>- A common interface to behavioural syllable classification methods

Reply:
How about:

- A common interface to prevalent pose estimation tools, and other machine-learning based object detection methods.

(avoids the repetition of "common" within sentence)

- A common interface to methods for behavioural segmentation and classification

(In my mind "behavioural syllable classification" sounds a bit off because there are often two distinct tasks: one is decomposing the behaviour into "syllables" (often called "action segmentation"), the other assigns "labels" to the actions (walking, sleeping etc.). They are often done jointly, as the first is kind of pre-requisite for the second.

Reply:
Sounds good. Updated.